### PR TITLE
fixing FE calls

### PR DESCRIPTION
**Pull request by jake-wickstrom-lb**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -89,15 +89,16 @@ app.post("/firebase-token", jsonParser, cors(), async function (req, res){
...
   try {
     // Basic input validation
-    validator.checkIsPresent(["length-minutes", "target-muscle-group"], req.body, "body", true);
+    validator.checkIsPresent(["length-minutes", "target-muscle-group"], req.query, "query", true);
 
     res.json(await workoutService.generateWorkoutPlan(
         JSON.parse(req.params.userId),
-        parseInt(req.body["length-minutes"], 10),
-        parseInt(req.body["target-muscle-group"], 10),
+        parseInt(req.query["length-minutes"], 10),
```
Comment (danaharlos) [inline server/server.js:96]: When something like this is changed I do have to refactor the backend tests for that API call - which is fine, but if you notice anymore that need to be refactored the sooner the better

File: server/server.js [hunk 2]
```diff
@@ -89,15 +89,16 @@ app.post("/firebase-token", jsonParser, cors(), async function (req, res){
...
+        parseInt(req.query["target-muscle-group"], 10),
         dbConfig
     ));
   } catch(ex) {
+    console.log(ex);
```
Comment (danaharlos) [inline server/server.js:101]: We haven't been posting console log messages for the rest of our API calls, we just send the exception in the response. For the sake of consistency we may want to just keep doing this, but if you see value then keep it

**Review by danaharlos — APPROVED** [review]
Looks good, just a couple comments

**Review by zfrantzen — APPROVED** [review]

File: server/server.js (hunk 1 above)
Comment (jake-wickstrom-lb) [reply server/server.js:96]: Agree. I haven't done an audit of all requests but GET requests are not allowed to have body parameters, if there are other get requests that do they should be changed to query params

File: server/server.js (hunk 2 above)
Comment (jake-wickstrom-lb) [reply server/server.js]: removed console.log

**Pull request merged**